src/express/api.js

```diff
@@ -38,9 +43,27 @@ class API {
     return this._load(`/category`, {params: {count}});
   }
 
+  async getOneCategory(id) {
```

Comment:
```suggestion
  getOneCategory(id) {
```
так будет одно и то же поведение. может подправить везде